hexrd/ui/main_window.py

```diff
@@ -626,3 +627,12 @@ def on_action_transform_detectors_triggered(self):
         self.image_mode_widget.reset_masking()
         td = TransformDialog(self.ui).exec_()
         self.image_mode_widget.reset_masking(mask_state)
+
+    def update_indexing_menu(self):
+        enabled = False
+        image_series_dict = HexrdConfig().imageseries_dict
```

Comment:
Can this line be changed to these two lines instead?

```python
image_series_dict = ImageLoadManager().unaggregated_images
image_series_dict = HexrdConfig().imageseries_dict if image_series_dict is None else image_series_dict
```

Basically, we want to use unaggregated images if possible. But if it is `None`, then we use the `HexrdConfig().imageseries_dict` instead.

Reply:
Will do. just fyi: (a) I wasn't sure that the ImageLoadManager contents were persistent, and (b) I think that Hexconfig.imageseries_dict is loaded the way you've prescribed. (but I might have extrapolated too much.)

Reply:
Yeah, it's a singleton as well. Its contents are persistent.

The complication is that, if the images were aggregated, we don't want to use `HexrdConfig().imageseries_dict`, but we want to use `HexrdConfig().unaggregated_images` instead. This follows the logic [here](https://github.com/HEXRD/hexrdgui/blob/5168755ee256cca68c9917793a67a134a29d7a12/hexrd/ui/indexing/create_config.py#L39-L44).